applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1629,6 +1688,20 @@ should_require_pin(MailboxJObj) ->
         'false' -> kzd_voicemail_box:pin_required(MailboxJObj)
     end.
 
+-spec allow_ff_rw(kz_json:object()) -> boolean().
+allow_ff_rw(MailboxJObj) ->
+    case ?DEFAULT_ALLOW_FF_RW of
```

Comment:
I think the logic is not quite right here. The `kapps_config` flag should be used to allow the voicemail box's setting to be used (and not as an allowed-by-default setting like it is this way). I would rewrite this to:

```erlang
case ?DEFAULT_ALLOW_FF_RW of
    'true' -> kzd_voicemail_box:allow_ff_rw(MailboxJObj);
    'false' -> 'false'
end
```

So if `kapps_config` returns `false` the ff/rw feature is not available on this system, full stop (retaining backwards compat). If the system operator turns that config flag to `true` then the voicemail box's setting is used (defaulting to `false` to preserve existing functionality).

Does that make sense?

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)

Reply:
@jamesaimonetti There was some confusion here.  We actually only want to have to enable the feature at the kapps_config level.  We removed any VMBox level config.

Reply:
@jamesaimonetti We have updated as requested.